test/util/grpctest/mocks.go

```diff
@@ -93,7 +97,7 @@ func StartMockServer(t *testing.T, service *MockTxService) (*grpc.ClientConn, fu
 	stop := func() {
```

Comment:
by t.Cleanup, I meant that here instead of returning the function, you can do t.Cleanup func right here in line.